src/pages/Referral/util.ts

```diff
@@ -0,0 +1,19 @@
+export const getFirebaseLinkDomain = () => {
+  if (
+    typeof window !== 'undefined' &&
+    (window as any).FIREBASE_LINK_DOMAIN !== undefined
+  ) {
+    return (window as any).FIREBASE_LINK_DOMAIN
```

Comment:
we're making `return`s in virtually every branch here, do we really need all these `else`s n stuff? get's hard to read

Reply:
I tried to mirror the design found in `utils/apolloClient.ts::getGiraffeEndpoint`, but I'm open for suggestions on how to express it of course 🙂

Reply:
Blaming that file i feel a bit silly hehe 😬 imo not one of my strongest pieces of code. imo it should be refactored into something like

```ts
  if (
    typeof window !== 'undefined' &&
    (window as any).FIREBASE_LINK_DOMAIN !== undefined
  ) {
    return (window as any).FIREBASE_LINK_DOMAIN
 }

  if (process.env.NODE_ENV === 'development') {
    return process.env.FIREBASE_LINK_DOMAIN || 'https://hedvigtest.page.link'
  } 

  if (
    process.env.NODE_ENV !== 'development' &&
    process.env.FIREBASE_LINK_DOMAIN
  ) {
    return process.env.FIREBASE_LINK_DOMAIN
  }

  throw Error(
    'Unable to find firebase link domain - specify with envvar `FIREBASE_LINK_DOMAIN`',
  )
```

Reply:
Far prettier - I'm a huge fan of the early return over else's as well 😉